scripts/chroot_reconst.sh

```diff
@@ -73,13 +73,25 @@ echo "deb https://download.sublimetext.com/ apt/stable/" | tee /etc/apt/sources.
 apt-get -qq update
 apt-get -qq -y install sublime-text
 
-# Installing related softwares
-chmod +x ./*.sh
-./*.sh
-
 # Installing Firefox
 apt-get -qq -y install firefox
 
+# Installing Chrome
+if [[ $(getconf LONG_BIT) = "64" ]]
```

Comment:
@rpotter12 It would be much better if you check the variable `$arch` to check the processor version that is defined in the build script
We need least verbose logs, hence no  need to echo statements and add quiet parameter to the wget command again for the same reason